### PR TITLE
math-magicians

### PR DESCRIPTION
**Pull request by Melaku05**
-Create react application
-setup linters
-setup github workflow

### CODE REVIEW

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: INVALID CODE REVIEW REQUEST 🛑 🛑

Hi @Melaku05,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

### You have submitted a project that does not have a README

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# MATH-MAGICIANS
+
```
Comment (uwadonat) [inline README.md]: - [ ] You have duplicate files in you project. you can remove some.
Screenshot from 2022-04-26 12-38-50


- [ ] node_modules should not be pushed to your github. You can check this [link](https://git-scm.com/docs/gitignore) to know why and how to do it.

**Review by uwadonat — CHANGES_REQUESTED** [review]
Hi **[Melaku05](https://github.com/Melaku05)**, :hand: 

Good job so far! :mechanical_arm: 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>
### To highlight! :+1: 
- Your readme file is very descriptive ✔️
- No linter errors ✔️
almost


## Status:Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
### Hi @Melaku05, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONAL] As pointed out by the previous code reviewer, which is a best practice, please make sure you don't push `node_modules` to your repository for it makes your repository heavy and crowded, the `package.json` file is enough for install any dependencies required to run your project. Please kindly add it to your `.gitignore`  file and delete the one you already have in your repository. Delete `node_modules` directory in your repository, and then add it to your `.gitignore file` locally.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**